### ISSUE TITLE
Ore harvester ignores blacklisted ores

### ISSUE THREAD

**Issue by LuciusV** [Dunkmania101/SpatialHarvesters#37]
When I use Resource disabler key and add resource to ore harvester blacklist, it still can give that ore, although I get the message about ore added to machine blacklist. 
Played as client connected to game opened to LAN on another machine, so I was not the server hosting played
Mod version: 1.16.5-0.11.5

**Comment by MSandro** [comment]
same issue here, Fabric Version 0.11.5 I tried to blacklist ores in the config too, but without success. Adding Ores is also currently not possible. I tried to add quartz ore.
```
commonConfig {
    "blacklist_bios"=[]
    "blacklist_bios_mod"=[
        botania
    ]
    "blacklist_loot"=[]
    "blacklist_loot_mod"=[]
    "blacklist_mobs"=[]
    "blacklist_mobs_mod"=[]
    "blacklist_ores"=[
	[
            "modern_industrialization",
            "copper_ore"
        ],
	[
            "modern_industrialization",
            "tin_ore"
        ],
	[
            "modern_industrialization",
            "salt_ore"
        ],
	[
            modern_industrialization",
            "silver_ore"
        ],
	[
            "modern_industrialization",
            "lead_ore"
        ]
    ]
    "blacklist_ores_mod"=[]
    "blacklist_soils"=[]
    "blacklist_soils_mod"=[]
    "blacklist_stones"=[]
    "blacklist_stones_mod"=[]
    "block_hardness"=5
    "block_resistance"=6
    "custom_bio_tags"=[
        [
            c,
            veggies
        ],
        [
            c,
            vegetables
        ],
        [
            c,
            leather
        ],
        [
            c,
            feathers
        ],
        [
            c,
            seeds
        ],
        [
            c,
            dyes
        ],
        [
            c,
            bones
        ],
        [
            c,
            "wooden_rods"
        ],
        [
            minecraft,
            "small_flowers"
        ],
        [
            minecraft,
            logs
        ],
        [
            minecraft,
            planks
        ],
        [
            minecraft,
            saplings
        ],
        [
            minecraft,
            leaves
        ]
    ]
    "custom_bios"=[]
    "custom_loot_tables"=[
        [
            minecraft,
            "chests/simple_dungeon"
        ]
    ]
    "custom_mob_drops"=[
        [
            [
                minecraft,
                "ender_dragon"
            ],
            [
                minecraft,
                elytra
            ]
        ],
        [
            [
                minecraft,
                "ender_dragon"
            ],
            [
                minecraft,
                "dragon_head"
            ]
        ],
        [
            [
                minecraft,
                "ender_dragon"
            ],
            [
                minecraft,
                "dragon_egg"
            ]
        ],
        [
            [
                minecraft,
                "ender_dragon"
            ],
            [
                minecraft,
                "dragon_breath"
            ]
        ]
    ]
    "custom_ore_tags"=[
        [
            c,
            ores
        ],
        [
            minecraft,
            "nether_quartz_ore"
        ]
    ]
    "custom_ores"=[
        [
            appliedenergistics2,
            "quartz_ore"
        ],
        [
            appliedenergistics2,
            "charged_quartz_ore"
        ],
        [
            rftoolsbase,
            "dimensionalshard_overworld"
        ],
        [
            rftoolsbase,
            "dimensionalshard_nether"
        ],
        [
            rftoolsbase,
            "dimensionalshard_end"
        ],
        [
            rhodonite,
            "block_ore_fluorite"
        ],
        [
            rhodonite,
            "block_ore_rhodonite"
        ],
        [
            "exp_ore",
            "block_exp_ore"
        ],
        [
            rockcandy,
            "candy_ore"
        ],
        [
            lightestlamp,
            "boron_ore"
        ],
        [
            funores,
            "zombie_ore"
        ],
        [
            funores,
            "blaze_ore"
        ],
        [
            funores,
            "ghast_ore"
        ],
        [
            funores,
            "magma_cube_ore"
        ],
        [
            funores,
            "wither_skeleton_ore"
        ],
        [
            funores,
            "zombie_pigman_ore"
        ],
        [
            funores,
            "salmon_ore"
        ],
        [
            funores,
            "squid_ore"
        ],
        [
            funores,
            "cod_ore"
        ],
        [
            funores,
            "sheep_ore"
        ],
        [
            funores,
            "rabbit_ore"
        ],
        [
            funores,
            "pig_ore"
        ],
        [
            funores,
            "cow_ore"
        ],
        [
            funores,
            "chicken_ore"
        ],
        [
            funores,
            "bat_ore"
        ],
        [
            funores,
            "witch_ore"
        ],
        [
            funores,
            "spider_ore"
        ],
        [
            funores,
            "slime_ore"
        ],
        [
            funores,
            "phantom_ore"
        ],
        [
            funores,
            "skeleton_ore"
        ],
        [
            funores,
            "enderman_ore"
        ],
        [
            funores,
            "guardian_ore"
        ],
        [
            funores,
            "pufferfish_ore"
        ],
        [
            funores,
            "creeper_ore"
        ],
        [
            gobber2,
            "gobber2_ore"
        ],
        [
            gobber2,
            "gobber2_ore_nether"
        ],
        [
            gobber2,
            "gobber2_ore_end"
        ]
    ]
    "custom_soil_tags"=[
        [
            c,
            dirt
        ],
        [
            c,
            sand
        ],
        [
            c,
            gravel
        ]
    ]
    "custom_soils"=[
        [
            minecraft,
            clay
        ]
    ]
    "custom_stone_tags"=[
        [
            c,
            cobblestone
        ],
        [
            c,
            sandstone
        ],
        [
            c,
            "end_stones"
        ],
        [
            c,
            netherrack
        ]
    ]
    "custom_stones"=[]
    "dimensional_applicator_amplifier"=1
    "dimensional_applicator_capacity_multiplier"=100
    "dimensional_applicator_divisor"=10.0
    "dimensional_applicator_duration"=220
    "dimensional_applicator_is_beacon_effect"=true
    "dimensional_applicator_price"=1000.0
    "dimensional_applicator_show_icon"=false
    "dimensional_applicator_show_particles"=false
    "enable_bio_harvesters"=true
    "enable_chunk_loader"=true
    "enable_dark_mob_harvester"=true
    "enable_dimensional_applicator"=true
    "enable_heat_generator"=true
    "enable_ore_harvesters"=true
    "enable_soil_harvesters"=true
    "enable_specific_mob_harvester"=true
    "enable_stone_harvesters"=true
    "harvester_capacity_multiplier"=100
    "harvester_shard_chance"=75
    "heat_generator_capacity_multiplier"=100
    "key_break_speed_multiplier"=10
    "machine_light_level"=7
    "min_tier_bios"=[]
    "min_tier_ores"=[]
    "min_tier_soils"=[]
    "min_tier_stones"=[]
    "mob_harvester_mob_shard_chance"=25
    "price_bio_1"=400.0
    "price_bio_2"=600.0
    "price_bio_3"=1000.0
    "price_bio_4"=1500.0
    "price_bio_5"=4000.0
    "price_bio_6"=7000.0
    "price_bio_7"=7500.0
    "price_bio_8"=8000.0
    "price_dark_mob"=8000.0
    "price_loot"=8000.0
    "price_ore_1"=400.0
    "price_ore_2"=600.0
    "price_ore_3"=1000.0
    "price_ore_4"=1500.0
    "price_ore_5"=4000.0
    "price_ore_6"=7000.0
    "price_ore_7"=7500.0
    "price_ore_8"=8000.0
    "price_soil_1"=400.0
    "price_soil_2"=600.0
    "price_soil_3"=1000.0
    "price_soil_4"=1500.0
    "price_soil_5"=4000.0
    "price_soil_6"=7000.0
    "price_soil_7"=7500.0
    "price_soil_8"=8000.0
    "price_specific_mob"=8000.0
    "price_stone_1"=400.0
    "price_stone_2"=600.0
    "price_stone_3"=1000.0
    "price_stone_4"=1500.0
    "price_stone_5"=4000.0
    "price_stone_6"=7000.0
    "price_stone_7"=7500.0
    "price_stone_8"=8000.0
    "speed_bio_1"=400
    "speed_bio_2"=350
    "speed_bio_3"=300
    "speed_bio_4"=250
    "speed_bio_5"=200
    "speed_bio_6"=150
    "speed_bio_7"=100
    "speed_bio_8"=5
    "speed_dark_mob"=50
    "speed_heat_generator"=3
    "speed_loot"=50
    "speed_ore_1"=400
    "speed_ore_2"=350
    "speed_ore_3"=300
    "speed_ore_4"=250
    "speed_ore_5"=200
    "speed_ore_6"=150
    "speed_ore_7"=100
    "speed_ore_8"=5
    "speed_soil_1"=400
    "speed_soil_2"=350
    "speed_soil_3"=300
    "speed_soil_4"=250
    "speed_soil_5"=200
    "speed_soil_6"=150
    "speed_soil_7"=100
    "speed_soil_8"=5
    "speed_specific_mob"=50
    "speed_stone_1"=400
    "speed_stone_2"=350
    "speed_stone_3"=300
    "speed_stone_4"=250
    "speed_stone_5"=200
    "speed_stone_6"=150
    "speed_stone_7"=100
    "speed_stone_8"=5
}

```

**Comment by Dunkmania101** [comment]
I think I found the problem! It's one I'd had earlier with the Mob Key. Essentially, I was trying to get the registry name of the item but was instead getting its translation key. For example: "minecraft:dirt" vs "item.minecraft.dirt". I've used the search in folder tool to find all occurrences of the text "getTranslationKey" and have fixed (hopefully :) all places that should be using something else. I'll release a fix in 0.11.6.
